pom.xml

```diff
@@ -145,6 +145,7 @@
     <flink.clients.artifactId>flink-clients</flink.clients.artifactId>
     <flink.connector.kafka.artifactId>flink-connector-kafka</flink.connector.kafka.artifactId>
     <flink.hadoop.compatibility.artifactId>flink-hadoop-compatibility_2.12</flink.hadoop.compatibility.artifactId>
+    <rocksdbjni.version>5.17.2</rocksdbjni.version>
```

Comment:
although no risk, we should avoid having irrelevant change mixed in a patch that meant for other issue.